City-of-Pori/pori-kada
# PORI-182 Main image and styling

To test:

1. drush fra -y && drush cc all
2. Go to edit any page in local.visitpori.fi and add it liftup image if there isn't one already.
3. Check the "Show main image on the page" checkbox active
4. Save the node and make sure that on the desktop view the content goes over the main image and the main image is visible.
5. Go do the same in local.pori.fi. The functionality should be same EXCEPT that the content won't go over the main image like it does in visitpori side.
6. Make sure the main image is not visible if the checkbox "Show main image on the page" is not checked.
